pyproject.toml

```
@@ -14,20 +14,1472 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
+[build-system]
+requires = [
+    "hatchling",
+    "gitpython",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "apache-airflow"
+dynamic = ["version"]
+
+description = " Programmatically author, schedule and monitor data pipelines"
+readme = { file = "generated/PYPI_README.md", content-type = "text/markdown" }
+license = { file = "LICENSE", content-type = "text/plain" }
+requires-python = "~=3.8"
+authors = [
+    { name = "Apache Software Foundation", email = "dev@airflow.apache.org" },
+]
+maintainers = [
+    {name="Apache Software Foundation", email="dev@airflow.apache.org"},
```

INconsistent spacing compared to above.
```suggestion
    {name = "Apache Software Foundation", email = "dev@airflow.apache.org"},
```